scripts/populate_lambda_ssm_parameters.py

```diff
@@ -69,15 +69,16 @@ def get_elasticsearch_endpoint():
     return domain_info['DomainStatus']['Endpoint']
 
 def get_admin_user_emails():
-    secret_id = "{}/{}/{}".format(
+    secret_base = "{}/{}/".format(
         os.environ['DSS_SECRETS_STORE'],
-        os.environ['DSS_DEPLOYMENT_STAGE'],
-        os.environ['GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME'],
-    )
-    resp = boto3.client("secretsmanager").get_secret_value(SecretId=secret_id)
+        os.environ['DSS_DEPLOYMENT_STAGE'])
+
+    gcp_service_id = secret_base + os.environ['GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME']
+    admin_service_id = secret_base + 'admin_user_email.json'
+    resp = boto3.client("secretsmanager").get_secret_value(SecretId=gcp_service_id)
     gcp_service_account_email = json.loads(resp['SecretString'])['client_email']
-    admin_user_emails = [email for email in os.environ['ADMIN_USER_EMAILS'].strip().split(",")
```

Comment:
All occurrences of `ADMIN_USER_EMAILS` outside of this file should be removed.

Reply:
added support function within [dss/util/__init__.py](dss/util/__init__.py), removed references